website/settings/defaults.py

```diff
@@ -2052,6 +2058,8 @@ class CeleryConfig:
 PLOS_METRICS_BASE_FOLDER = None
 PLOS_METRICS_INITIAL_FILE_DOWNLOAD_URL = None
 PLOS_METRICS_OSF_TOKEN = None
+SPAM_METRICS_BASE_FOLDER = None
```

Comment:
Make sure this gets into the release playbook

Reply:
These are now spurious